### PR TITLE
Blog Post: Interactive Book New Avatar

### PR DESCRIPTION
**Pull request by Shivansh2407**
This is a Blog post regarding the launch of the all new revamped Interactive Book!

### CODE REVIEW

**Comment by Shivansh2407** [comment]
We should merge this after we merge GSOD2020 branch in the Interactive Book Repository. Do let me know your views on this Blog @satu0king @tachyons.

File: content/posts/Interactive-book-new-avatar.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: "Interactive Book's New Avatar!"
+date: 2021-05-26T11:12:38+05:30
+draft: false
+author: Shivansh Srivastava
+type: post
+url: "/posts/Interactive-Book-New-Avatar/"
```
Comment (tachyons) [inline content/posts/Interactive-book-new-avatar.md:7]:
```suggestion
url: "/posts/interactive-book-revamp/"
```

File: content/posts/Interactive-book-new-avatar.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+---
+
+
+We are excited 🎊 to launch of all new Interactive Book, completely redesigned and full of content related to digital logic design! The idea behind the revamp is to create a user-friendly browsing experience for the students, professors and learners across the globe. We hope you experience enriched learning!
```
Comment (tachyons) [inline content/posts/Interactive-book-new-avatar.md:11]: Significant change is in the content, design is mostly the same as before

File: content/posts/Interactive-book-new-avatar.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+Daniel completely rebuilt the website, from updating the current structure of the Interactive Book, adding circuits and diagrams, to finally creating new matter and improvising the previous ideas.
+
+The Idea of pop quizzes was implemented and some bugs like Dark Mode, Deployment Issues etc. were fixed by our Community Members and Contributors before the final launch!
+
+We would also like to thank our amazing contributors at Circuitverse who gave their precious time and exceptional energy to make this site what it is!
+
+## Wish to chip-in?
+Go ahead, Clone the [Repository](https://github.com/CircuitVerse/Interactive-Book/) and propose features/enhancements or fix bugs🐞!
+
+Make sure you join Circuitverse [Slack](https://join.slack.com/t/circuitverse-team/shared_invite/enQtNjc4MzcyNDE5OTA3LTdjYTM5NjFiZWZlZGI2MmU1MmYzYzczNmZlZDg5MjYxYmQ4ODRjMjQxM2UyMWI5ODUzODQzMDU2ZDEzNjI4NmE) as well for discussions related to Circuitverse projects and Digital Logic Design!
```
Comment (tachyons) [inline content/posts/Interactive-book-new-avatar.md:44]: Use circuitverse.com/slack link as full url can expire at any time

File: content/posts/Interactive-book-new-avatar.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+As contributors and maintainers of this project, and in the interest of fostering an open and welcoming community, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
+
+## What's popping? 🤔
+
+- Rearranged current content intro new tree structure
+- Created content which is not covered in the earlier version. You will get to learn more from the enhanced content along with detailed diagrams.
+- Included Logic Circuits designed with Circuitverse for all the topics for a better understanding! 
+- Enhanced content
+- Pop Quizzes for each Topic (Learn More)
```
Comment (tachyons) [inline content/posts/Interactive-book-new-avatar.md:27]: Contribution Guide
Documentation is now explicitly under CC-SA license

File: content/author/shivansh-srivastava.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
++++
+title = "Shivansh Srivastava"
+image = "/images/authors/Shivansh2407.jpeg"
+description = "Full Stack developer"
++++
+
+[Shivansh2407](https://github.com/Shivansh2407)
+
+Google Summer of Code 2021 Mentor • Google Season of Docs 2020 Mentor
```
Comment (tachyons) [inline content/author/shivansh-srivastava.md:9]: You can add more about you here, not just things related to the Circuitverse

File: content/posts/Interactive-book-new-avatar.md (hunk 3 above)
Comment (Shivansh2407) [reply content/posts/Interactive-book-new-avatar.md:44]: http://www.circuitverse.org/slack

**Comment by tachyons** [comment]
@nitin10s

File: content/posts/Interactive-book-new-avatar.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+![Interactive Book Launch](/images/Banner-IB-Launch.png)
+
+## What is [Interactive Book](https://learn.circuitverse.org)?
+
+The aim of the project is to create an online interactive guide for digital logic design. The primary goal is to develop an open source book with quality content that teaches digital logic design. It will enable students to learn digital design by interacting with circuits, truth table, and other interactive elements as they proceed through the book. The professors and students all over the world can read and contribute to the same.
+
+As contributors and maintainers of this project, and in the interest of fostering an open and welcoming community, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
+
```
Comment (nitin10s) [inline content/posts/Interactive-book-new-avatar.md:20]: "As contributors..." line can be shifted to the Wish to chip in section.

Comment (nitin10s) [inline content/posts/Interactive-book-new-avatar.md:20]: "As contributors..." line can be shifted to the Wish to chip in section.

File: content/posts/Interactive-book-new-avatar.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+I was offered an opportunity by [Satvik](https://github.com/satu0king) and [Aboobacker](https://github.com/tachyons) to mentor the projects listed by Circuitverse for Google Season Of Docs 2020, and after carefully reviewing all the proposals we chose [Daniel Vicente Lühr Sierra](https://github.com/danielluehr) for **"CircuitVerse Interactive Book Consolidation and Improvements"** Project.
+
+Daniel completely rebuilt the website, from updating the current structure of the Interactive Book, adding circuits and diagrams, to finally creating new matter and improvising the previous ideas.
+
+The Idea of pop quizzes was implemented and some bugs like Dark Mode, Deployment Issues etc. were fixed by our Community Members and Contributors before the final launch!
+
+We would also like to thank our amazing contributors at Circuitverse who gave their precious time and exceptional energy to make this site what it is!
+
+## Wish to chip-in?
+Go ahead, Clone the [Repository](https://github.com/CircuitVerse/Interactive-Book/) and propose features/enhancements or fix bugs🐞!
```
Comment (nitin10s) [inline content/posts/Interactive-book-new-avatar.md:44]: Wish to contribute? seems a simpler heading

File: content/posts/Interactive-book-new-avatar.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+- Detailed Contributing Guidelines
+- The book is now licensed under the [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/) and copyrighted by contributors to [CircuitVerse](https://circuitverse.org/)
+
+Want to know more? View the final report of the project [here](https://blog.circuitverse.org/posts/gsod2020-cvib-final-report-dvls/).
+
+## Get to know the brilliant minds behind the curtains!🥁
+
+I was offered an opportunity by [Satvik](https://github.com/satu0king) and [Aboobacker](https://github.com/tachyons) to mentor the projects listed by Circuitverse for Google Season Of Docs 2020, and after carefully reviewing all the proposals we chose [Daniel Vicente Lühr Sierra](https://github.com/danielluehr) for **"CircuitVerse Interactive Book Consolidation and Improvements"** Project.
+
+Daniel completely rebuilt the website, from updating the current structure of the Interactive Book, adding circuits and diagrams, to finally creating new matter and improvising the previous ideas.
```
Comment (nitin10s) [inline content/posts/Interactive-book-new-avatar.md:37]: A line introducing Daniel and mentioning his experience would be a great addition

**Review by nitin10s — COMMENTED** [review]
Everything else looks great!

**Comment by Shivansh2407** [comment]
Done! @tachyons @nitin10s @satu0king :+1:

**Comment by satu0king** [comment]
Looks great!

**Comment by satu0king** [comment]
@ShreyaPrasad1209 can you make a social media post on this?

**Pull request merged**